contracts/scripts/e2e/bridge/test_standard_bridge_withdraw_eth.ts

```diff
@@ -47,25 +44,28 @@ async function main() {
 
   const blockNumber = txWithLogs.blockNumber;
 
-  let lastConfirmedBlockNumber = 0;
   let assertionWasCreated = false;
-  let assertionId;
+  let assertionId: number | undefined = undefined;
+  let lastConfirmedBlockNum: number | undefined = undefined;
 
   inbox.on(
     inbox.filters.TxBatchAppended(),
-    async (batchNumber, previousInboxSize, inboxSize, event) => {
-      const tx = await event.getTransaction();
-      lastConfirmedBlockNumber = await getLastBlockNumber(tx.data);
+    (event) => {
+      console.log(`TxBatchAppended blockNum: ${event.blockNumber}`)
     }
   );
 
-  rollup.on(rollup.filters.AssertionConfirmed(), (id) => {
+  rollup.on(rollup.filters.AssertionConfirmed(), async (id: BigNumber) => {
```

Comment:
I see this is a copypasta with L129-133 of `contracts/scripts/e2e/bridge/test_standard_bridge_erc20.ts`. Is there a good way of cleaning this up?

Reply:
I think once we have all the `e2e` tests passing, there's a lot of cleanup and refactoring we could do